roles/collector_pd/tasks/main.yml

```diff
@@ -13,6 +13,12 @@
 - include_tasks: collect_log.yml
 - include_tasks: collect_config.yml
 
+- name: collect PD information
+  shell: python {{ collector_dir }}/scripts/tidb-insight/insight.py --output={{ pd_log_dir }} --alias={{ inventory_hostname }} --pdctl --pd-host={{ pd_host }} --pd-port={{ pd_client_port }}
```

Comment:
pd_host is not defined in this role, I found I've defined a  service_host variable in main.yml, see https://github.com/pingcap/tidb-ansible/blob/master/roles/collector_pd/tasks/main.yml#L5.